ansible/pbTestScripts/buildJDK.sh

```diff
@@ -6,13 +6,18 @@ if [ -z "${WORKSPACE:-}" ]; then
 		WORKSPACE=$HOME
 fi
 [[ ! -d "$WORKSPACE/openjdk-build" ]] && git clone https://github.com/adoptopenjdk/openjdk-build $WORKSPACE/openjdk-build
+
+# If detected GCC version is <7, set it to 7 
```

Comment:
If this is required, something has gone wrong. The logic in https://github.com/AdoptOpenJDK/openjdk-build/blob/master/build-farm/platform-specific-configurations/linux.sh sets `CC` and `CXX` to the correct value - hacking symlinks outside the playbook shouldn't be required.

Reply:
Looking at the logic of that - it only sets the correct value of `CC` and `CXX` when it's building JDK11+-J9, or when it's `s390x` and below JDK11-J9, it will set gcc to 4.8.

Reply:
Incorrect - the `s390x` at the start is overriden by this section:
`if [ "$JAVA_FEATURE_VERSION" -gt 10 ] || [ "${VARIANT}" == "${BUILD_VARIANT_OPENJ9}" ]; then`
which will take effect on all OpenJ9 versions.

Reply:
Ah my mistake.
I've just had a look at an U16 machine that had the playbook run on it, and gcc7 isn't installed to `/usr/local/gcc`. CentOS6 does however have gcc-7.3 installed to `/opt/rh/devtoolset-7/root/usr/bin` as the script you linked is looking for. 
On Ubuntu machines, it installs it to `/usr/bin/x86_64-linux-gnu-gcc-7`

Reply:
U16 should also by symlinked to `/usr/bin/gcc-7` (Which is used in the guard to stop https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/gcc_7/tasks/main.yml from installing it into /usr/local but as you say not in the environment file above. Have created https://github.com/AdoptOpenJDK/openjdk-build/pull/1446/files to resolve this - feel free to review that for me ;-)

Reply:
Reviewed and merged the above PR, therefore removed this block of code.